app/controllers/ember-community-survey-2017.js

```diff
@@ -391,118 +341,88 @@ const longLived = {
         15.8, // '3–6 Months',
         19.8, // '< 3 Months',
       ],
-      color: emberOrange,
     },
   ],
-};
+}).highchartsOptions;
 
-const serverLanguage = {
-  options: {
-    chart: {
-      type: 'column',
-    },
-    title: {
-      text: 'What is the primary language of your server-side stack?',
-    },
-    subtitle: {
-      text: '',
-    },
-    xAxis: {
-      type: 'category',
-    },
-    yAxis: {
-      title: {
-        text: 'Percent',
-      },
-    },
+const serverLanguage = new VerticalBarChart({
+  chart: {
+    categories: [
+      'Ruby',
+      'JavaScript',
+      'Java',
+      'Python',
+      'PHP',
+      '.NET',
+      'Elixir / Erlang',
+      'Go',
+      'Scala',
+      'TypeScript',
+      'C++',
+      'Other',
+    ],
```

Comment:
Let's use 2016, which has more items, as the source of truth for categories.

```suggestion
    categories: [
      'Ruby',
      'JavaScript',
      'Java',
      'PHP',
      'Python',
      '.NET',
      'Elixir / Erlang',
      'Go',
      'Scala',
      'TypeScript',
      'C++',
      'Rust',
      'Other',
    ],
```